shared/src/main/scala/squants/mass/MomentOfIntertia.scala

```diff
@@ -0,0 +1,73 @@
+package squants.mass
+
+import squants.motion.{AngularAcceleration, NewtonMeters, Torque}
+import squants.space.{Feet, Meters}
+import squants.{AbstractQuantityNumeric, Dimension, Length, PrimaryUnit, SiBaseUnit, StrictlyPositiveQuantity, UnitConverter, UnitOfMeasure}
+
+/**
+  *
+  * @author paxelord
+  * @since 0.6
+  *
+  * @param value Double
+  */
+final class MomentOfInertia private (val value: Double, val unit: MomentOfInertiaUnit)
+    extends StrictlyPositiveQuantity[MomentOfInertia](value){
+
+  def dimension = MomentOfInertia
+
+  def toKilogramsMetersSquared = to(KilogramsMetersSquared)
+  def toPoundsSquareFeet = to(PoundsSquareFeet)
+
+  def *(angularAcceleration: AngularAcceleration): Torque = {
+    val radiansPerSecondSquared = angularAcceleration.toRadiansPerSecondSquared
+
+    NewtonMeters(toKilogramsMetersSquared * radiansPerSecondSquared)
+  }
+
+  /**
+    * For a point mass with the given MomentOfInertia rotating with a center of
+    * rotation at the given radius, return the mass of the point mass
+    * @param radius distance to axis of rotation
+    * @return mass of point mass with given radius and MomentOfInertia
+    */
+  def atCenter(radius: Length): Mass = {
+    Kilograms(toKilogramsMetersSquared / radius.squared.toSquareMeters)
+  }
+}
+
+object MomentOfInertia extends Dimension[MomentOfInertia] {
+  private[mass] def apply[A](n: A, unit: MomentOfInertiaUnit)(implicit num: Numeric[A]) = new MomentOfInertia(num.toDouble(n), unit)
+  def apply = parse _
+  def name = "MomentOfInertia"
+  def primaryUnit = KilogramsMetersSquared
+  def siUnit = KilogramsMetersSquared
+  def units = Set(KilogramsMetersSquared, PoundsSquareFeet)
+}
+
+trait MomentOfInertiaUnit extends UnitOfMeasure[MomentOfInertia] with UnitConverter {
+  def apply[A](n: A)(implicit num: Numeric[A]) = {
+    MomentOfInertia(num.toDouble(n), this)
+  }
+}
+
+object KilogramsMetersSquared extends MomentOfInertiaUnit with PrimaryUnit with SiBaseUnit {
+  val symbol = Kilograms.symbol + "‧" + Meters.symbol + "²"
+}
+
+object PoundsSquareFeet extends MomentOfInertiaUnit {
+  val symbol = Pounds.symbol + "‧" + Feet.symbol + "²"
+  val conversionFactor = Pounds.conversionFactor * math.pow(Feet.conversionFactor, 2D)
+}
+
+object MomentOfInertiaConversions {
+  lazy val kilogramMetersSquared = KilogramsMetersSquared(1)
+  lazy val poundSquareFeet = PoundsSquareFeet(1)
+
+  implicit class MassConversions[A](n: A)(implicit num: Numeric[A]) {
```

Comment:
Just chiming in, @Paxelord and I are on the same robotics team, where we're using Squants. I think this can still be made into a value class, if the implicit is moved to the methods. So `MassConversions` (or `MomentofInertiaConversions` I guess) would only take a `val n`, and the `kilogramMetersSquared` and `poundSquareFeet` would take the implicit `Numeric`.